app/ide-desktop/lib/content-config/src/config.json

```diff
@@ -1,5 +1,13 @@
 {
   "options": {
+    "authentication": {
+      "value": false,
+      "description": "Determines whether user authentication is enabled. This option is disregarded if the application is executed in the cloud."
+    },
+    "newDashboard": {
+      "value": false,
+      "description": "Determines whether the dashboard (containing the directory listing) is enabled."
```

Comment:
I don't think "new-dashboard" should be top-level option. We have there a group for things in the app that we are testing, like the new component browser. This option should land there.

Reply:
ok sounds good. Moving it to `featurePreview` group then